DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -66,17 +66,17 @@ public IEnumerable<CourseStatistics> GetCentreSpecificCourseStatistics(int centr
         )
         {
             return courseDataService.GetDelegateCoursesInfo(delegateId)
+                .Where(info => info.CustomisationCentreId == centreId)

```

Comment:
I just want to double check the changes do actually do what we want. 

With this new CustomisationCentreId check, an admin looking at the View Delegate page will not be able to see any AllCentre courses that the delegate is enrolled on that are defined at a different centre. An admin at the centre the AllCentre course is defined at will not be able to see this either, as the delegate is registered at a different centre. 

As such, I don't think any admin will be naturally be able to follow a standard journey in the View Delegate to see/edit the progress details of a delegate for an AllCentre course. Is this the intended design? The service filter VerifyAdminUserCanAccessProgress does allow for AllCentre courses to be viewed and modified, so the user can still reach those pages by altering URLs, and also from the CourseDelegates page (I think).

Reply:
I've just seen 612, and it does indeed seem like the behaviour we want. As such, the course delegates and progress pages will probably need modifying as well. Though it still feels an odd choice to me.

Reply:
I think you are correct that they should probably be able to view the All Centres courses that the delegate is enrolled on. It's not like they are managing the courses (as noted when I split VerifyAdminUserCanAccessCourse for HEEDLS-655 on the internal slack channel).

Updated the service method to have this. (This means I really don't think the above point about combining the wheres should happen)